stage1/usr_from_coreos/coreos-common.mk

```diff
@@ -11,9 +11,13 @@ $(call setup-tmp-dir,CCN_TMPDIR)
 # systemd version in coreos image
 CCN_SYSTEMD_VERSION := v229
 # coreos image version
+ifeq ($(RKT_ACI_ARCH),aarch64)
+CCN_IMG_RELEASE := 1053.2.0
+else
 CCN_IMG_RELEASE := 1032.0.0
+endif
```

Comment:
Let's update to 1068.0.0 then, which is also the current base for the beta channel. I'm testing a bump to it right now for amd64, if everything is fine I'll ping back here with the PR.